pagopa/pn-frontend
# Bff dependency - Feat/pn 12392

## Short description
Updated bff dependency and made changes due to api updates.

## List of changes proposed in this pull request
- Updated consent actions (pa/pf/pg)
- Updated TermsOfService page (pa/pf/pg)
- Replaced all occurrences of channelType SERCQ with SERCQ_SEND 
- Call tos and privacy acceptance before service activation
- Updated tests

## How to test
Login in with pa/pf/pg -> check that the TOS/PRiVACY retrieve and acceptance work well
Login in with pf/pg -> check that the Digital Domicile works well
